drivers/adc/adc_lmp90xxx.c

```diff
@@ -107,8 +107,7 @@ LOG_MODULE_REGISTER(adc_lmp90xxx);
 #define LMP90XXX_HAS_DRDYB(config) (config->drdyb.port != NULL)
 
 struct lmp90xxx_config {
-	const struct device *spi_dev;
-	struct spi_config spi_cfg;
+	const struct spi_dt_spec bus;
```

Comment:
In general, specifying `const` qualifier to struct members should be avoided and the `const` qualification should be added during struct declaration instead (which, by the way, we already do).

Reply:
Fixed for all, in my head I was replicating `const struct device *dev`, but the semantics are different with pointers.